.travis.yml

```diff
@@ -57,10 +57,10 @@ script:
 - echo "TRAVIS_PULL_REQUEST_SLUG=${TRAVIS_PULL_REQUEST_SLUG}"
 - echo "TRAVIS_REPO_SLUG=${TRAVIS_REPO_SLUG}"
 - >-
-  if [ $JDKVER = jdk8-plus ] ; then
-    DOCKERIMAGE=mdernst/cf-$OS-$JDKVER
-  else
+  if [ $JDKVER = downstream || $JDKVER = checker-framework-inference ] ; then
```

Comment:
Instead of `JDKVER` should this line use `GROUP`?

Reply:
Ah, my fault, solved.